src/functions/deduplicateLeads.js

```diff
@@ -0,0 +1,24 @@
+const { writeLog } = require("./logDuplicates");
+
+const deduplicateLeads = (leadsArr) => {
+  return leadsArr
+    .slice()
+    .reverse()
```

Comment:
Why is the `reverse()` call necessary? It failed the tests when I ran the code without it.